service/services/eventDiscoveryService/install/offline/awsmsk/build.sh

```diff
@@ -1,3 +1,3 @@
-cp ../../../target/event-discovery-agent-1.0-SNAPSHOT.jar .
-docker build --build-arg JAR_FILE=event-discovery-agent-1.0-SNAPSHOT.jar --tag event-discovery-agent:1.0 .
-rm ./event-discovery-agent-1.0-SNAPSHOT.jar
+cp ../../../target/event-discovery-agent-app-1.0-SNAPSHOT.jar .
```

Comment:
This is unrelated to the reported error, but it will allow the easy creation of a docker container from the built jar.